src/publishProject/publishProjectWebViewController.ts

```diff
@@ -125,11 +130,23 @@ export class PublishProjectWebViewController extends FormWebviewController<
         this.updateState();
 
         // Fetch Docker tags for the container image dropdown
-        if (projectTargetVersion) {
-            const tagComponent =
-                this.state.formComponents[constants.PublishFormFields.ContainerImageTag];
-            if (tagComponent) {
-                await loadDockerTags(projectTargetVersion, tagComponent, this.state.formState);
+        // Use the deployment UI function with target version filtering
+        const tagComponent = this.state.formComponents[PublishFormFields.ContainerImageTag];
+        if (tagComponent) {
+            try {
```

Comment:
This should end up being removed to just use `-latest` for all, but waiting PM confirmation.